### ISSUE TITLE
Blink Example

### ISSUE THREAD

**Issue by folny** [PXforever/BT_IAP_STM32F4#1]
Good day

I like your project BT Bootloader just not sure how to write an application that is uploaded to STM using it, you can add the same code for example LED blink thank you.

**Comment by PXforever** [comment]
Yes, You're right. I will give a application example soon. And complete more detail for how To use this code. And 
I want make this project Perfect.,So if you have any interesting, you can join it.

**Comment by folny** [comment]
It will be great when the code will work perfectly I found that the code does not work with FTDI Usb chip need to change \xff to \n or \r at least in my case it was not possible. You can look at this project is very similar to yours but is for the STM32F1 series.

https://github.com/havenxie/stm32-iap-uart-boot

**Comment by PXforever** [comment]
I have read this code that you sent me, yes it has samilar principle , this is flash programing(at ST official site has given reference manual). and I use the CH340 TTLtoUSB chip, maybe it can work normally in my PC. finally, it a trueth that this code has less compatiblity.
manual url:
https://www.st.com/content/ccc/resource/technical/document/programming_manual/5b/ca/8d/83/56/7f/40/08/CD00228163.pdf/files/CD00228163.pdf/jcr:content/translations/en.CD00228163.pdf

**Comment by folny** [comment]
If you have FTDI Usb at home try not to work me. But it does not matter I would like it to work mainly through BT module Nrf51 or Nrf52.

**Comment by PXforever** [comment]
In a new push, the demo is added to the project, and a new readme updated. this app is an i2c driver. 
I don't have FTDI, and BT module is HC05, it the only one that I can find in my studio.
